internal/email/users.go

```diff
@@ -348,3 +350,107 @@ func (s *Service) SendMergedUpdate(UserName string, UserEmail string) error {
 
 	return nil
 }
+
+// SendTeamInvite sends the team invite email to unregistered user
+func (s *Service) SendTeamInvite(TeamName string, UserEmail string, InviteID string) error {
+	emailBody, err := s.generateBody(
+		hermes.Body{
+			Name: "",
+			Intros: []string{
+				"Register to join your team on Thunderdome!",
+			},
+			Actions: []hermes.Action{
+				{
+					Instructions: fmt.Sprintf(
+						"Please register for Thunderdome using the following link (expires in 24 hours) to join the %s Team.",
+						TeamName),
+					Button: hermes.Button{
+						Color: "#22BC66",
+						Text:  "Register Account",
+						Link:  s.Config.AppURL + "register/team/" + InviteID,
+					},
+				},
+				{
+					Instructions: "Need help, or have questions? Visit our Github page",
+					Button: hermes.Button{
+						Text: "Github Repo",
+						Link: "https://github.com/StevenWeathers/thunderdome-planning-poker/",
```

Comment:
the repo should probably be an app constant.

Reply:
Updated in main.